staging/ledger_state.json

```diff
@@ -45,5 +45,5 @@
       ]
     }
   ],
-  "hash": "ca17a2c678d52849228b7f44f31ffdc4cdf1111b718d97043d7b313793a53a64"
+  "hash": "112598f5158d0a8f95a89dfd3000899ad617999a8445a65032ca81dbc15cf745"
```

Comment:
Why did the hash change?

Reply:
Different hashing technique between merk and fmerk?  I'm not 100% sure.